src/main/knn/KnnGraphTester.java

```diff
@@ -343,6 +358,11 @@ private void run(String... args) throws Exception {
     }
     if (indexPath == null) {
       indexPath = Paths.get(formatIndexPath(docVectorsPath)); // derive index path
+      log("Index Path = %s", indexPath);
+    }
+    if (parentJoin && !reindex && !isParentJoinIndex(indexPath)) {
```

Comment:
`== false` instead of single-character `!`?

Reply:
Done